src/pow.cpp

```diff
@@ -38,7 +39,7 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
         return nProofOfWorkLimit;
 
     // right at fork
-    else if(isForkBlock(nHeight) && !isForkBlock(nHeight - params.nPowAveragingWindow))
```

Comment:
Why has this been removed?

Reply:
It was removed to keep difficulty constant during fork period.